articles/migration/updating_versions.md

```diff
@@ -0,0 +1,66 @@
+
+1. Open Visual Studio 2022
+2. In the lanuch window, choose the *Continue without code* option at the bottom on the right.
+3. From the top menu choose, *Extensions > Manage Extensions*  to open the *Extension Manager* panel.
+4. Click the *Updates* tab and choose *MonoGame Framework C# project templates* and update it to the latest version.
+
+If prompted to close Visual Studio to finish the update, do so now to continue installing the update.
+
+## dotnet CLI
+Develoeprs using the dotnet CLI with environments such as Visual Studio Code, JetBrains Rider, or other editors, can exceute the following command in a command prompt/terminal to update the templates
+
+```sh
+dotnet new uninstall MonoGame.Templates.CSharp
+dotnet new install MonoGame.Templates.CSharp
+```
+
+The templates must be uninstalled first before the updated tempaltes are installed; otherwise, you will see warnings about template naming conflicts between the two versions.
+
+Alternatively, you can perform `dotnet new update` which will update all templates installed to their most current version avaialble.  However, this may affect other templates you have installed that you may not wish to update, it's an all or nothing command.
+
+# Update Existing Projects
+Developers wishing to update their current project to use a new release version only need to update the *target framework*, *NuGet package verions*, and *dotnet tools* version. Doing this dpeends on if you are using Visual Studio 2022 or another development environment
+
+## Visual Studio 2022
+Open your existing project in Visual Studio 2022 and perform the following
+1. Right-click on the project in the *Solution Explorer* panel and select *Manage NuGet Packages...".
+2. In the NuGet Packages Manager window, click the *Updates* tab.
+3. Select the MonoGame Framework packages in the list on the left, then click the *Update* button for each one on the right.
+
+## Manually Editing .csproj File
+Alternatively, regardless of the development environment, developers can manually edit the *.csproj* file for their project to update to the newest MonoGame release.  To do this
+1. Open your project's *.csproj* file
+2. Locate the `<PacakgeReference>` elements for any MonoGame referenced packages.
+3. Change the `Version=` attribute for each one to the current version of MonoGame (currently this is `3.8.2.1105`)
+4. Save the *.csproj* file.
+
+Once these changes are made, open a command prompt or terminal at the root of the project directory and enter the following commands
+
+```sh
+dotnet clean
+dotnet restore
+```
+
+## Update MGCB Editor
```

Comment:
Also put this at the top for existing projects.

Maybe have a SUbject headers to denote:

* Updating existing projects
* Updating environment for new projects

Reply:
Moved sections